app/src/main/java/com/example/atheneum/activities/BookInfoActivity.java

```diff
@@ -370,6 +410,96 @@ public void deleteBook(){
         }
     }
 
+    /**
+     * Attempt to get information from goodreads
+     */
+    public void getGoodreadsReviewInfo(long isbn) {
+        // show error instead if there is no internet connection
+        ConnectionChecker connectionChecker = new ConnectionChecker(this);
+        if (!connectionChecker.isNetworkConnected()) {
+            hideGoodreadsReview();
+            showGoodreadsReviewError("Ratings and reviews unavailable while offline. \n");
+            return;
+        }
+
+        goodreadsAvgRatingbar = findViewById(R.id.goodreadsAvgRatingBar);
+        getReviewsBtn = findViewById(R.id.gotoReviewsBtn);
+        String apiRequestURL = null;
+
+
+        // null check for ISBN
+        if (textIsbn.getText() != null && !textIsbn.getText().toString().equals("")) {
+            apiRequestURL = "https://www.goodreads.com/book/isbn/" + String.valueOf(isbn) +
```

Comment:
I am having issues receiving reviews for a lot of my books. I can receive the ratings (or so it seems) but I don't get reviews. Even though some of the books I have have thousands of reviews (like Deep Work by Cal Newport), try running the ISBN for my book 9781455586691 to see if you can see this issue too.

Reply:
Although, all my books have 4.5 ratings which is weird since for example Elements of Programming Interviews has a rating of 4.28 on Goodreads (https://www.goodreads.com/book/show/16253895-elements-of-programming-interviews)

Reply:
Fixed

Reply:
For future reference, this is an issue with the secondaryTint of the rating bar, which if not set to transparent, will still color in the other half of the star as the same color